README.md

```diff
@@ -1,3 +1,5 @@
+**NOTE**: this plugin is currently maintained at https://github.com/jenkinsci/ghprb-plugin
```

Comment:
Why this note?  GHPRB has a Jira project as well.

https://issues.jenkins-ci.org/browse/JENKINS-55793?jql=project%20%3D%20JENKINS%20AND%20status%20in%20(Open%2C%20%22In%20Progress%22%2C%20Reopened)%20AND%20component%20%3D%20ghprb-plugin%20ORDER%20BY%20created%20DESC

At this point it's hard to stop using either without a migration because there's many good reports in both locations.